docs/integrations/azure_pipelines.md

```diff
@@ -438,6 +438,13 @@ information. Since these tokens are sensitive, **care should be taken to protect
           GITHUB_TOKEN: $(GITHUB_PAT)
 ```
 
+### Exit Codes
+
+The Phylum analysis job will return a zero (0) exit code when it completes successfully and a non-zero code otherwise.
+The full and current list of exit codes, with explanations, is [documented here][exit_codes].
```

Comment:
```suggestion
The full and current list of exit codes is [documented here][exit_codes].
```

I feel like this is implied in documenting them.

Reply:
The change was made across all the integration docs.